src/Microsoft.AspNetCore.Mvc.Razor.Extensions/MvcRazorTemplateEngine.cs

```diff
@@ -23,6 +23,7 @@ public class MvcRazorTemplateEngine : RazorTemplateEngine
             RazorProject project)
             : base(engine, project)
         {
+            Options.ImportsFileName = "_ViewImports.cshtml";
```

Comment:
I guess this is going to break Mvc functionals. Could you get the PR for it ready?